R/utils.R

```diff
@@ -182,3 +182,117 @@ is_installed <- function(path_pkg = ".") {
 is_windows <- function() {
   identical(.Platform$OS.type, "windows")
 }
+
+#' Pin asset directory
```

Comment:
Can this function also handle files or only directories? Maybe it would be worthwhile to extend it (you don't have to, we can postpone it)

Reply:
Yeah I thought about that too, havent tested it yet though. Would make sense to extend it to files too though